### DISCUSSION TITLE
Speech translate for Uzbek language.

### DISCUSSION THREAD

**Discussion by zohirsharipov** [Dadangdut33/Speech-Translate#55] [General]
Hi Fauzan, 

Can I contact with you by email or whatsapp? I need this project for uzbek language. If you have  a profile on fiverr or upwork and other frilancing platform, please share your account for  the cooperation

**Comment by realtechspecs** [comment]
> Hi Fauzan,
> 
> Can I contact with you by email or whatsapp? I need this project for uzbek language. If you have a profile on fiverr or upwork and other frilancing platform, please share your account for the cooperation

What type of content do you want to translate?
